src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -36,6 +36,9 @@
     <PackageReference Include="Microsoft.ServiceHub.Client" />
     <PackageReference Include="Microsoft.VisualStudio.Debugger.Contracts" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\Compilers\Core\Portable\InternalUtilities\IsExternalInit.cs" Link="Utilities\IsExternalInit.cs" />
```

Comment:
seems like a bunch of projects were picking up this from the clasp project.  Had to add it in a few places now that its become a source package.